### DISCUSSION TITLE
carbon's documentation

### DISCUSSION THREAD

**Discussion by shivendra0712** [carbon-language/carbon-lang#2696] [Language design]
My name is Shivendra Patel, Currently  i am in 3rd Year Of B.Tech Computer Science with Specialization Artificial Intelligence Machine Learning. A Computer Science undergrad at LNCTS BHOPAL willing to learn and develop new skills and trying to improve the known for organization benefit. 

I would like to contribute on your project can you please suggest me resources.
So, i can learn technogies  that it used in my project.

I had my interest laid down on the project Creating documentation for Carbon source files.

The project details are:

Difficulty level: ____
Skills required/preferred: C++, HTML, MarkDown, language parsing
Project size: 350 hr

Generate documentation from Carbon source files. This might include:

Producing HTML and/or MarkDown output
Converting declarations for classes, functions, interfaces, and so on.
Extracting marked comments
Recognizing commenting conventions for parameters, returns, cross reference, markdown code snippets, and so on.
Cross referencing declarations, definitions, and possibly uses.
Continuous integration for deployment

And some other information about me.

-   I am good in C, C++, data structure and algoritham ,  HTML, CSS, Java Script, git,  github . 
-  I have completed Salesforce Developer Virtual Internship.

 Contect me:
[Email](patelshivendra817@gmail.com)
[linkedin](https://www.linkedin.com/in/shivendra-patel-31845b212)

  My Progress:
[Geeksforgeeks](https://auth.geeksforgeeks.org/user/shivendrapatel01250)
[Leetcode](https://leetcode.com/shivendrapatel01250/)

 I am honest, and other abilities, including problem-solving, critical thinking, creativity etc.

**Comment by shivendra0712** [comment]
Kindly check @jonmeow Sir then guide me. 
 Thank You

**Reply by jonmeow** [reply]
@shivendra0712 Proposals are only being accepted through the GSoC website; [here's Carbon's page](https://summerofcode.withgoogle.com/programs/2023/organizations/carbon-language). For more information, please read the [GSoC "How it Works"](https://summerofcode.withgoogle.com/how-it-works) and [Carbon's GSoC contributor guidelines](https://docs.google.com/document/d/1Ly3tD58IE57h_xsoZ8wsL8y4Mp89fg8J8qLN8sCPZ5k/preview?resourcekey=0-CLlGUVSc7DMhLPTABpmjRQ#). If you're trying to figure out how to contribute to Carbon, [CONTRIBUTING.md](https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md) is a good starting point.

**Reply by shivendra0712** [reply]
Okay sir
